<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE TS>
<TS version="2.1" language="it" sourcelanguage="en">
<context>
    <name>generals</name>
    <message>
        <location filename="../XTFLog_Checker_dock_panel.py" line="44"/>
        <source>Show errors</source>
        <translation>Mostra errori</translation>
    </message>
    <message>
        <location filename="../XTFLog_Checker_dock_panel.py" line="45"/>
        <source>Show warnings</source>
        <translation></translation>
    </message>
    <message>
        <location filename="../XTFLog_Checker_dialog.py" line="45"/>
        <source>Create layer</source>
        <translation>Creare layer</translation>
    </message>
    <message>
        <location filename="../XTFLog_Checker_dialog.py" line="48"/>
        <source>Cancel</source>
        <translation>Annulla</translation>
    </message>
    <message>
        <location filename="../XTFLog_Checker_dialog.py" line="51"/>
        <source>Show error log</source>
        <translation>Mostra registro errori</translation>
    </message>
    <message>
        <location filename="../XTFLog_Checker_dialog.py" line="52"/>
        <source>Upload xtf-log file</source>
        <translation>Scegliere percorso o URL di un file XTF-Error-Log</translation>
    </message>
    <message>
        <location filename="../XTFLog_Checker_dialog.py" line="53"/>
        <source>Show log for existing layer</source>
        <translation>Mostra il registro degli errori per un layer esistente</translation>
    </message>
    <message>
        <location filename="../XTFLog_Checker_dialog.py" line="54"/>
        <source>Only layers created with this plugin can be selected</source>
        <translation>È possibile selezionare solo i layer creati con questo plugin</translation>
    </message>
    <message>
        <location filename="../XTFLog_Checker_dialog.py" line="94"/>
        <source>No valid file</source>
        <translation>File non valido</translation>
    </message>
    <message>
        <location filename="../XTFLog_Checker_dialog.py" line="86"/>
        <source>Could not get a valid XTF-Log file from specified Url</source>
        <translation>Impossibile ottenere un file XTF-Log valido dall&apos;URL specificato</translation>
    </message>
    <message>
        <location filename="../XTFLog_Checker_dialog.py" line="94"/>
        <source>No valid XTF-Log file at specified Path</source>
        <translation>Nessun file XTF-Log valido nel percorso specificato</translation>
    </message>
    <message>
        <location filename="../XTFLog_Checker_dock_panel.py" line="48"/>
        <source>Error log</source>
        <translation>XTFLog-Checker registro errori</translation>
    </message>
    <message>
        <location filename="../XTFLog_Checker_dialog.py" line="155"/>
        <source>No Errors</source>
        <translation>Non ci sono errori</translation>
    </message>
    <message>
        <location filename="../XTFLog_Checker_dialog.py" line="155"/>
        <source>The selected XTF file contains no Ilivalidator-Errors, select another file.</source>
        <translation>Il file selezionato non contiene errori del tipo IliVErrors, selezionare un altro file.</translation>
    </message>
    <message>
        <location filename="../XTFLog_Checker_dialog.py" line="90"/>
        <source>Large file</source>
        <translation>File più grande di 5 MB</translation>
    </message>
    <message>
        <location filename="../XTFLog_Checker_dialog.py" line="90"/>
        <source>Processing of large XTF-Log files might take a while</source>
        <translation>L&apos;elaborazione di file di grandi dimensioni potrebbe richiedere del tempo</translation>
    </message>
</context>
</TS>
